File: SDKs/yodlee.com/1.1.0/python/src/sdk/models/operations/gettransactioncategories.py
import dataclasses
from typing import Optional
from ..shared import transactioncategoryresponse as shared_transactioncategoryresponse


@dataclasses.dataclass
class GetTransactionCategoriesResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    transaction_category_response: Optional[shared_transactioncategoryresponse.TransactionCategoryResponse] = dataclasses.field(default=None)
    

File: SDKs/yodlee.com/1.1.0/typescript/src/sdk/models/operations/gettransactioncategories.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetTransactionCategoriesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  transactionCategoryResponse?: shared.TransactionCategoryResponse;
}
